### ISSUE TITLE
Unresponsive script on first import in Z for Firefox, OS X

### ISSUE THREAD

**Issue by owcz** [zotero/zotero#883]
When syncing (or restoring) from the server on a new install, near the end of the transfer, I get this error:
![screenshot 2015-12-15 11 51 28](https://cloud.githubusercontent.com/assets/778011/11818915/95a2188a-a322-11e5-9b00-63e54e39b8eb.png)
When I click continue, the sync wraps up and shows one of my subfolders rather than My Library (so I was under the impression that only a few files synced). I don't get the same error when importing in Standalone on a fresh install and data directory.

**Comment by owcz** [comment]
I played around with this a bit more, did a few fresh installs, and think this is specific to Z for Firefox. So even with a fresh install of Zotero for Firefox (OS X), I only set Zotero sync (no automatic sync, no file syncing checkboxes), I after the loading bar, I get the unresponsive script notice and ~31 files—the same issue. My workaround is to uninstall/ignore the Firefox install, load Standalone with its own data directory and to sync with the above settings (not automatic, no file syncing checkboxes). It very quickly loaded the full directory and I was able to add my other settings back afterwards. Now I can point Z for Firefox to use the Standalone's data directory, but I would have preferred to have avoided the standalone altogether.
